### PR TITLE
Adding extra auth validation cases

### PR DESCRIPTION
**Pull request by gapra-msft**

### CODE REVIEW

File: azure-pipeline-templates/build.yml [hunk 1]
```diff
@@ -114,6 +114,8 @@ steps:
       echo "\"block-key\"": "\"$(AZTEST_BLOCK_KEY)\"", >> $cnfFile
       echo "\"adls-key\"": "\"$(AZTEST_ADLS_KEY)\"", >> $cnfFile
       echo "\"block-sas\"": "\"$(AZTEST_BLOCK_SAS)\"", >> $cnfFile
+      echo "\"block-cont-sas-ubn-18\"": "\"$(AZTEST_BLOCK_CONT_SAS_UBN_18)\"", >> $cnfFile
+      echo "\"block-cont-sas-ubn-20\"": "\"$(AZTEST_BLOCK_CONT_SAS_UBN_20)\"", >> $cnfFile
```
Comment (vibhansa-msft) [inline azure-pipeline-templates/build.yml:118]: UBN 22 is also there with similar naming convention.

File: blobfuse2-code-coverage.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ jobs:
           stressParallel: 3
         Ubuntu-20:
           imageName: 'ubuntu-20.04'
-          containerName: 'test-cnt-coverage'
+          containerName: 'test-cnt-ubn-20'
```
Comment (vibhansa-msft) [inline blobfuse2-code-coverage.yaml:23]: This was explicitly done to avoid same container being used by both nightly and coverage.

**Review by vibhansa-msft — APPROVED** [review]
Left few minor comments.

File: blobfuse2-code-coverage.yaml (hunk 2 above)
Comment (gapra-msft) [reply blobfuse2-code-coverage.yaml:23]: the likelihood of them being run at the same time is kinda low right?

**Review by souravgupta-msft — COMMENTED** [review]
The binary file **azure-storage-fuse** is checked in. Can you add this file to .gitignore?

**Review by souravgupta-msft — APPROVED** [review]

File: blobfuse2-code-coverage.yaml (hunk 2 above)
Comment (vibhansa-msft) [reply blobfuse2-code-coverage.yaml:23]: Code coverage for now is on demand basis but once we make it run periodically then likelihood might increase. Any specific reason you wanted to change the name here ?

Comment (gapra-msft) [reply blobfuse2-code-coverage.yaml:23]: mostly cause it runs container sas tests now

Comment (gapra-msft) [reply blobfuse2-code-coverage.yaml:23]: also why do we allow the ubn18 container to overlap

File: azure-pipeline-templates/build.yml (hunk 1 above)
Comment (gapra-msft) [reply azure-pipeline-templates/build.yml:118]: I dont see a SAS for Block UBN22 in the Nightly Variables

**Review by tasherif-msft — APPROVED** [review]

File: blobfuse2-code-coverage.yaml (hunk 2 above)
Comment (gapra-msft) [reply blobfuse2-code-coverage.yaml:21]: it also makes managing the SASs hard since we need one per container

Comment (tasherif-msft) [reply blobfuse2-code-coverage.yaml:21]: I think we should minimize maintaining SASs and we can just rerun in the odd case

**Pull request merged**